shared/clinical-code-sets/medications/barbituates/1/barbituates.ctv3.txt

```diff
@@ -73,4 +73,12 @@ dn6x.	Methylphenobarbitone 30mg tablet
 dna2.	Mysoline 250mg/5mL oral suspension
 dna1.	Mysoline 250mg tablet
 d31f.	Soneryl 100mg tablet
+dna3.	MYSOLINE 50mg tablets
+dna3.	MYSOLINE 50mg tablets
+X78EQ	Fetal primidone syndrome
```

Comment:
Remove "fetal primidone syndrome" as this sounds like a diagnosis rather than a medication